cli/exchange/service.go

```diff
@@ -563,3 +603,58 @@ func ServiceRemovePolicy(org string, credToUse string, service string, force boo
 func ServiceNewPolicy() {
 	policy.New()
 }
+
+// Get the Kubernetes operator yaml archive from the cluster deployment string and save it to a file
+func SaveOpYamlToFile(sId string, clusterDeployment string, filePath string) {
+	// get message printer
+	msgPrinter := i18n.GetMessagePrinter()
+
+	archiveData := GetOpYamlArchiveFromClusterDepl(clusterDeployment)
+
+	fileName := filePath
+	if info, err := os.Stat(filePath); err == nil && info.IsDir() {
+		_, id := cliutils.TrimOrg("", sId) // remove the org part
+		fileName = filepath.Join(filePath, id+"_operator_yaml.tar.gz")
+	}
+
+	if _, err := os.Stat(fileName); err == nil {
+		cliutils.Fatal(cliutils.CLI_INPUT_ERROR, msgPrinter.Sprintf("File %v already exist. Please specify a different file name.", fileName))
+	}
+
+	file, err := os.Create(fileName)
+	if err != nil {
+		cliutils.Fatal(cliutils.INTERNAL_ERROR, msgPrinter.Sprintf("Failed to create file %v. %v", fileName, err))
+	}
+	defer file.Close()
+
+	if _, err := file.Write(archiveData); err != nil {
+		cliutils.Fatal(cliutils.INTERNAL_ERROR, msgPrinter.Sprintf("Failed to save the Kube operator yaml archive data to file %v.", fileName))
+	}
+
+	msgPrinter.Printf("The Kube operator yaml archive is saved to file %v", fileName)
```

Comment:
Change "The Kube operator yaml" to "The clusterDeployment operator yaml"

Reply:
done